Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.6
```

Comment:
Oh no not for now. I guess it could be added in it but installing python3.5 could be a PITA in terms of waiting for it to install.

If anything this is just a nice to have.